ui/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -56,7 +56,7 @@ export default class SelectHardware extends Component {
 
   connect = async () => {
     if (this.state.selectedDevice) {
-      if (this.state.selectedDevice === 'trezor') {
+      if (this.state.selectedDevice === 'trezor' && window.navigator.usb) {
```

Comment:
can you throw in a comment about why this check is necessary? something about firefox support and maybe even link to https://mozilla.github.io/standards-positions/#webusb ?

Reply:
I have added a comment in 93cfd69fda